lib/app/modules/settings/views/weather_api.dart

```diff
@@ -6,7 +6,7 @@ import 'package:ultimate_alarm_clock/app/utils/constants.dart';
 import 'package:ultimate_alarm_clock/app/utils/utils.dart';
 import 'package:url_launcher/url_launcher_string.dart';
 
-class WeatherApi extends StatelessWidget {
```

Comment:
Why did you convert this to a stateful widget? We're using GetX for state management.

Reply:
Actually, I'm newbie and don't know much about Flutter.

I apologize for that.